src/__tests__/cat.test.ts

```diff
@@ -184,6 +218,61 @@ for (const format of ['symbolic', 'semantic'] as Array<'symbolic' | 'semantic'>)
       }
     });
 
+    it('should create a prior distribution with default parameters', () => {
+      const cat = new Cat();
+      expect(cat.priorDist).toBeDefined();
+      expect(cat.priorPar).toBeDefined();
+    });
+
+    it('should create a normal prior distribution with custom priorDist and priorPar', () => {
+      const cat = new Cat({ priorDist: 'norm', priorPar: [2, 0.5] });
+      expect(cat.priorDist).toBe('norm');
+      expect(cat.priorPar).toEqual([2, 0.5]);
+    });
+
+    it('should throw an error for invalid priorDist', () => {
+      expect(() => {
+        new Cat({ priorDist: 'invalid' as any, priorPar: [0, 1] });
```

Comment:
If you introduce the `priorDistType` union type that I recommended in `cat.ts`, then this line will have to change to something like

```suggestion
        new Cat({ priorDist: 'invalid' as unknown as priorDistType, priorPar: [0, 1] });
```